api/models/categoryModel.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+const CategorySchema = mongoose.Schema({
+    title: {
+        type: String,
+        required: [true, 'Please provide title.']
+    },
+    createdOn: {
+        type: String,
+        required: true,
+        default: new Date(),
+    }
+});
+
+module.exports = mongoose.model('categories', CategorySchema);
```

Comment:
Typically for the collection name the nomenclature I'm familiar with is using a Capital letter and singular 'Category'.